packages/in_app_purchase/example/README.md

```diff
@@ -1,8 +1,84 @@
-# in_app_purchase_example
+# In App Purchase Example
 
-Demonstrates how to use the in_app_purchase plugin.
+Demonstrates how to use the In App Purchase (IAP) Plugin.
 
 ## Getting Started
 
-For help getting started with Flutter, view our online
-[documentation](https://flutter.io/).
+WARNING: This plugin and example are not ready to be used. Right now the example
+app doesn't actually demonstrate any IAP functionality yet. Follow
+[flutter/flutter#9591](https://github.com/flutter/flutter/issues/9591) for more
+updates.
+
+There's a significant amount of setup required for testing in app purchases
+successfully, including registering new app IDs and store entries to use for
+testing in both the Play Developer Console and App Store Connect. Both Google
+Play and the App Store require developers to configure an app with in-app items
+for purchase to call their in-app-purchase APIs.
+
+### Android
+
+1. Create a new app in the [Play Developer
```

Comment:
These instructions are awesome, we should probably have something similar in the main README.md doc (doesn't have to happen in this PR)

Reply:
I added the basic intro paragraph to the main doc and had it point to here for more detailed example setup instructions.